docusaurus/docs/dev-docs/installation/cli.md

```diff
@@ -103,6 +103,10 @@ Follow the steps below to create a new Strapi project, being sure to use the app
 
 Once all questions have been answered, the script will start creating the Strapi project.
 
+:::caution
+If the browser didn't respond you might need to disable any razer processes running in the background.
```

Comment:
Is this specific to Windows? If yes, and I believe it is, we should mention it.

Reply:
I was using Windows 11 when it happened, but I'm not sure how it'll behave on other operating systems.

Reply:
By the way, are you mentioning processes for this [Razer](https://www.razer.com/)?

Reply:
Yes, this Razer. Usually, when Razer Synapse is installed, multiple Razer Chroma/Synapse processes run in the background without even launching the Razer Synapse/Chroma.

Reply:
OK, I see. So, I'm sorry, but this is way too specific of a use case and absolutely not related to Strapi itself, so this shouldn't be featured in the official Strapi docs. Feel free to create a post in the [forum](forum.strapi.io) or tell users in the [Discord forum](discord.strapi.io).
Thank you for your understanding. I will now close this PR.